src/main/resources/assets/storagedrawers/lang/it_it.json

```diff
@@ -0,0 +1,119 @@
+    "block.storagedrawers.birch_trim": "Compensato di Betulla",
+    "block.storagedrawers.jungle_full_drawers_1": "Drawer della giungla 1x1",
+    "block.storagedrawers.jungle_full_drawers_2": "Drawer della giungla 1x2",
+    "block.storagedrawers.jungle_full_drawers_4": "Drawer della giungla 2x2",
+    "block.storagedrawers.jungle_half_drawers_1": "Mezzo Drawer della giungla 1x1",
+    "block.storagedrawers.jungle_half_drawers_2": "Mezzo Drawer della giungla 1x2",
+    "block.storagedrawers.jungle_half_drawers_4": "Mezzo Drawer della giungla 2x2",
+    "block.storagedrawers.jungle_trim": "Compensato della Giungla",
+    "block.storagedrawers.acacia_full_drawers_1": "Drawer di acacia 1x1",
+    "block.storagedrawers.acacia_full_drawers_2": "Drawer della acacia 1x2",
+    "block.storagedrawers.acacia_full_drawers_4": "Drawer della acacia 2x2",
+    "block.storagedrawers.acacia_half_drawers_1": "Mezzo Drawer della acacia 1x1",
+    "block.storagedrawers.acacia_half_drawers_2": "Mezzo Drawer della acacia 1x2",
+    "block.storagedrawers.acacia_half_drawers_4": "Mezzo Drawer della acacia 2x2",
+    "block.storagedrawers.acacia_trim": "Compensato di Acacia",
+    "block.storagedrawers.dark_oak_full_drawers_1": "Drawer di quercia scura 1x1",
+    "block.storagedrawers.dark_oak_full_drawers_2": "Drawer di quercia scura 1x2",
+    "block.storagedrawers.dark_oak_full_drawers_4": "Drawer di quercia scura 2x2",
+    "block.storagedrawers.dark_oak_half_drawers_1": "Mezzo Drawer di quercia scura 1x1",
+    "block.storagedrawers.dark_oak_half_drawers_2": "Mezzo Drawer di quercia scura 1x2",
+    "block.storagedrawers.dark_oak_half_drawers_4": "Mezzo Drawer di quercia scura 2x2",
+    "block.storagedrawers.dark_oak_trim": "Compensato di Quercia Oscura",
+    "block.storagedrawers.crimson_full_drawers_1": "Drawer Cremisi 1x1",
+    "block.storagedrawers.crimson_full_drawers_2": "Drawer Cremisi 1x2",
+    "block.storagedrawers.crimson_full_drawers_4": "Drawer Cremisi 2x2",
+    "block.storagedrawers.crimson_half_drawers_1": "Mezzo Drawer Cremisi 1x1",
+    "block.storagedrawers.crimson_half_drawers_2": "Mezzo Drawer Cremisi 1x2",
+    "block.storagedrawers.crimson_half_drawers_4": "Mezzo Drawer Cremisi 2x2",
+    "block.storagedrawers.crimson_trim": "Compensato Cremisi",
+    "block.storagedrawers.warped_full_drawers_1": "Drawer Distorto 1x1",
+    "block.storagedrawers.warped_full_drawers_2": "Drawer Distorto 1x2",
+    "block.storagedrawers.warped_full_drawers_4": "Drawer Distorto 2x2",
+    "block.storagedrawers.warped_half_drawers_1": "Mezzo Drawer Distorto 1x1",
+    "block.storagedrawers.warped_half_drawers_2": "Mezzo Drawer Distorto 1x2",
+    "block.storagedrawers.warped_half_drawers_4": "Mezzo Drawer Distorto 2x2",
+    "block.storagedrawers.warped_trim": "Compensato Distorto",
+    "block.storagedrawers.compacting_drawers_3": "Drawer di Compattazione",
+    "block.storagedrawers.controller": "Controllore dei Drawer",
```

Comment:
Maybe "Regolatore" is a better translation instead of "Controllore". The dictionary says "Regolatore" is the right translation for machines: https://www.wordreference.com/enit/controller

Reply:
You're right, i should change "Controllore" but i think "Gestore" is more appropriate in this context.

Reply:
Yeah, it's more appropriate!